packages/applications-service-api/__tests__/unit/utils/representation.mapper.test.js

```diff
@@ -0,0 +1,23 @@
+const { mapBackOfficeRepresentationToApi } = require('../../../src/utils/representation.mapper');
+const {
+	REPRESENTATION_BACKOFFICE_RESPONSE,
+	REPRESENTATION_BACKOFFICE_DATA
+} = require('../../__data__/representation');
+const { SERVICE_USERS_BACKOFFICE_DATA } = require('../../__data__/serviceUser');
+const { BACK_OFFICE_DB_DOCUMENTS } = require('../../__data__/documents');
+describe('representation.mapper', () => {
+	describe('mapBackOfficeRepresentationToApi', () => {
```

Comment:
a test for the case where `representativeId` / the `representative` param is null would be good, as that will be a common scenario